composer.json

```diff
@@ -1,17 +1,22 @@
 {
     "name": "heureka/overenozakazniky",
-    "description": "Heureka overeno service API",
+    "description": "Heureka overeno zakazniky service API",
     "authors": [
         {
             "name": "Heureka.cz",
             "email": "podpora@heureka.cz"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "php": ">=5.1.0"
+        "php": ">=5.4.0"
```

Comment:
namespaces jsou od PHP v 5.3.0, ale nevím jak zbytek kódu, aby tam něco nevadilo.. Když někdo kdo má práva na repo. nastaví trigger na Travis, tak to můžem zkusit Travisem ;-) (kopni v Lib. do někoho ať to pls nastaví)